logsmanagement/logsmanag_config.c

```diff
@@ -0,0 +1,1367 @@
+                    snprintfz(path, FILENAME_MAX, "%s/error.log", get_log_dir());
+                    if(access(path, R_OK)) {
+                        collector_error("[%s]: 'Netdata_error.log' path (%s) invalid, unknown or needs permissions", 
+                            p_file_info->chartname, path);
+                        return p_file_info_destroy(p_file_info);
+                    } else p_file_info->filename = strdupz(path);
+                } else if(!strcasecmp(p_file_info->chartname, "Netdata_fluentbit.log")){
+                    if(access(p_flb_srvc_config->log_path, R_OK)){
+                        collector_error("[%s]: Netdata_fluentbit.log path (%s) invalid, unknown or needs permissions", 
+                            p_file_info->chartname, p_flb_srvc_config->log_path);
+                        return p_file_info_destroy(p_file_info);
+                    } else p_file_info->filename = strdupz(p_flb_srvc_config->log_path);
+                } else if(!strcasecmp(p_file_info->chartname, "Auth.log_tail")){
+                    const char * const auth_path_default[] = {
+                        "/var/log/auth.log",
+                        NULL
+                    };
+                    int i = 0;
+                    while(auth_path_default[i] && access(auth_path_default[i], R_OK)){i++;};
+                    if(!auth_path_default[i]){
+                        collector_error("[%s]: auth.log path invalid, unknown or needs permissions", p_file_info->chartname);
+                        return p_file_info_destroy(p_file_info);
+                    } else p_file_info->filename = strdupz(auth_path_default[i]);
+                } else if(!strcasecmp(p_file_info->chartname, "syslog_tail")){
+                    const char * const syslog_path_default[] = {
+                        "/var/log/syslog",   /* Debian, Ubuntu */
+                        "/var/log/messages", /* RHEL, Red Hat, CentOS, Fedora */
+                        NULL
+                    };
+                    int i = 0;
+                    while(syslog_path_default[i] && access(syslog_path_default[i], R_OK)){i++;};
+                    if(!syslog_path_default[i]){
+                        collector_error("[%s]: syslog path invalid, unknown or needs permissions", p_file_info->chartname);
+                        return p_file_info_destroy(p_file_info);
+                    } else p_file_info->filename = strdupz(syslog_path_default[i]);
+                }
+                break;
+            case FLB_WEB_LOG:
+                if(!strcasecmp(p_file_info->chartname, "Apache_access.log")){
+                    const char * const apache_access_path_default[] = {
+                        "/var/log/apache/access.log",
+                        "/var/log/apache2/access.log",  /* Debian and derivatives, Alpine */
+                        "/var/log/apache2/access_log",  /* Gentoo ? */
+                        "/var/log/httpd/access_log",    /* RHEL and derivatives */
```

Comment:
In reality, this is the default Apache log [file](https://httpd.apache.org/docs/2.4/logs.html), and it does not have relationship with RHEL.

Reply:
What's the recommended change here?

Reply:
I would suggest either to remove the comment or to change for `Apache default configuration file`.